src/Components/Navigation.jsx

```diff
@@ -75,17 +88,24 @@ const Navigation = () => {
                     <Box style={{ flexGrow: 1 }} />
 
                     {/* desktop menu */}
-                    <Box display={{ xs: "none", sm: "none", md: "flex" }}>
+                    <Box display={{ xs: "none", sm: "none", md: "none", lg: "flex" }}>
                         <ButtonGroup>
-                            {pages.map((page, idx) => (
-                                <NavButton key={idx} page={page} />
+                            {headerPages.map((page, idx) => (
+                                page.name !== "Access Calculator" ? <NavButton key={idx} page={page} /> : null
```

Comment:
Can we use a constant here instead of a raw string?

Reply:
Hi Katie! I've been working on this and am a little stuck on how to execute on that. I've been digging into typescript to figure it out but can you give me any additional context that will help move me in the right direction? Here is the sitemap that the map is iterating over:

<img width="495" alt="Screen Shot 2022-10-18 at 4 32 28 PM" src="https://user-images.githubusercontent.com/91432595/196564131-26f77709-e29f-47bd-aa21-3b2949173ff6.png">

Reply:
sorry forgot to tag @stuttskl

Reply:
Hey, I'm so sorry I missed this comment. GitHub goes notoriously overboard with emails regarding PRs. I will be better about following up on comments, so my apologies. Tagging also helps so thank you for doing that : ) So, I was thinking something like creating a custom type similar to how we do this in `CalculatorPageTypes.ts` and an example would be the `ProgressBar` file. You can see how in our `switch` statement our different cases are using the custom type `SectionName`. Let me know if this makes sense!

Reply:
@erluebbers

Reply:
@stuttskl Hi Katie, I made the change! I ended up adding the enumerable as a key in the Page type. I realized I could have just set the enum to be the names of the pages and do page.name !== PageId.WHATEVER but I was thinking that it may be helpful in the future. You can let me know if that sounds better and I can roll it back.